tests/resources/VCH-Util.robot

```diff
@@ -187,7 +187,8 @@ Gather Logs From Test Server
     Remove File  vic-admin-cookies
     ${out}=  Run  govc datastore.download %{VCH-NAME}/vmware.log %{VCH-NAME}-vmware.log
     Should Contain  ${out}  OK
-    ${out}=  Run  sshpass -p %{TEST_PASSWORD} scp %{TEST_USERNAME}@%{TEST_URL}:/var/log/vmkernel.log ./vmkernel.log
+    ${out}=  Run  sshpass -p %{TEST_PASSWORD} scp -o StrictHostKeyChecking\=no %{TEST_USERNAME}@%{TEST_URL}:/var/log/vmkernel.log ./vmkernel.log
```

Comment:
I left the sshpass, which will continue to work for esxi/vcenters where we still have ssh enabled with username/password e.g for nimbus and I assume nightly builds. Also for people running tests against their own environments.